open-build-service/notqmail-permissions.easy

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
A note to the effect that these `notqmail-permissions.*` apply only to OpenSUSE Enterprise rpms, and not other rpms.

Reply:
This has been addressed by commit #[679054c](https://github.com/notqmail/notqmail/pull/76/commits/679054c6130f8c5ee012714270234bf55cc921d7).